### ISSUE TITLE
Comcut break if no Commercials

### ISSUE THREAD

**Issue by arrmo** [BrettSheleski/comchap#19]
Hi,

It seems like if processing a shorter file, with no commercials => then things break. Or is it just me?

Thanks!

**Comment by BrettSheleski** [comment]
Could you be more specific of how "things break"?

**Comment by arrmo** [comment]
Sure! Sorry ... :-(. It seems that then the edl file is empty - that is expected. But then no cut file is generated. That seems OK, but a script no longer sees an output file then.

Or is it just me?

Thanks!

**Comment by BrettSheleski** [comment]
Still unsure what you're trying to say.

Are you specifying an output file via the command line to comcut?  Something like:

```
comcut /path/to/inputfile /path/to/outputfile
```

And since `inputfile` doesnt contain any commercials it is not writing to `outputfile`?

**Comment by arrmo** [comment]
Yep, exactly. And as I recall it's erroring out (i.e. returning a fail error code) - but I could be wrong on this one (sorry, remote right now, can't check ... :().

Thanks!

**Comment by BrettSheleski** [comment]
Gotcha.  This should be relatively easy to fix whenever I find time.

**Comment by arrmo** [comment]
Thanks! Definitely no biggie.
